mirage/serializers/crate-owner-invitation.js

```diff
@@ -24,7 +24,9 @@ export default BaseSerializer.extend({
     delete hash.id;
     delete hash.token;
 
-    hash.crate_id = Number(hash.crate_id);
+    // TODO: Check this further, `crate_id` are strings in the current fixtures
+    // when attempting to parse into number we will get `NaN`s here.
+    // hash.crate_id = Number(hash.crate_id);
```

Comment:
This is causing some trouble when retrieving crates by its ID, given that crates IDs are actually strings with the name of such crate.